deposit/src/main/java/edu/unc/lib/deposit/validate/VirusScanJob.java

```diff
@@ -134,17 +135,15 @@ public void runJob() {
 									+ result.getException()
 											.getLocalizedMessage());
 				case PASSED:
-					Element ev2 = getEventLog().logEvent(
-							PremisEventLogger.Type.VIRUS_CHECK,
-							"File passed pre-ingest scan for viruses.", href.getKey(),
-							ContentModelHelper.Datastream.DATA_FILE.getName());
-					PremisEventLogger.addSoftwareAgent(ev2, "ClamAV", version);
-					PremisEventLogger.addDetailedOutcome(ev2, "success", null,
-							null);
-					appendDepositEvent(href.getKey(), ev2);
-						scannedObjects++;
+					premisEvent = premisEventBuilder.addSoftwareAgent("ClamAV", version)
+						.addEventDetail("success")
```

Comment:
I believe the message here should be "File passed pre-ingest scan for viruses."?